src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/Instance/Authenticator.cs

```diff
@@ -42,7 +42,9 @@ internal class Authenticator
     {
         private const string TenantlessTenantName = "Common";
 
-        private bool updatedFromTemplate;
+        private bool _updatedFromTemplate;
+
+        internal static Regex TenantNameRegex = new Regex(Regex.Escape(TenantlessTenantName), RegexOptions.IgnoreCase | RegexOptions.CultureInvariant);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Change the visibility of 'TenantNameRegex' or make it 'const' or 'readonly'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://iddp-sonarqube:9000/coding_rules#rule_key=csharpsquid%3AS2223)

Reply:
make it private

Reply:
done.